templates/conventions/recapitulatif.html

```diff
@@ -60,50 +60,44 @@
-                        {% include "conventions/recapitulatif/foyer_residence_logements.html" with target_url='conventions:avenant_foyer_residence_logements' lot=convention.lot logements=convention.lot.logements_import_ordered %}
-                        {% include "conventions/recapitulatif/collectif.html" with target_url='conventions:avenant_collectif' lot=convention.lot %}
-                        {% include "conventions/recapitulatif/logements.html" with target_url='conventions:avenant_logements' lot=convention.lot logements=convention.lot.logements_import_ordered logements_sans_loyer=convention.lot.logements_sans_loyer_import_ordered logements_corrigee=convention.lot.logements_corrigee_import_ordered logements_corrigee_sans_loyer=convention.lot.logements_corrigee_sans_loyer_import_ordered %}
-                        {% include "conventions/recapitulatif/annexes.html" with target_url='conventions:avenant_annexes' annexes=convention.lot.annexes.all lot=convention.lot %}
+                        {% include "conventions/recapitulatif/foyer_residence_logements.html" with target_url='conventions:avenant_foyer_residence_logements' lot=convention.lot logements=convention.lot.logements_import_ordered checked=True %}
+                        {% include "conventions/recapitulatif/logements.html" with target_url='conventions:avenant_logements' lot=convention.lot logements=convention.lot.logements_import_ordered logements_sans_loyer=convention.lot.logements_sans_loyer_import_ordered logements_corrigee=convention.lot.logements_corrigee_import_ordered logements_corrigee_sans_loyer=convention.lot.logements_corrigee_sans_loyer_import_ordered checked=True %}
+                    {% else %}
+                        {% include "conventions/recapitulatif/foyer_residence_logements.html" with target_url='conventions:avenant_foyer_residence_logements' lot=convention.lot logements=convention.lot.logements_import_ordered checked=False %}
+                        {% include "conventions/recapitulatif/logements.html" with target_url='conventions:avenant_logements' lot=convention.lot logements=convention.lot.logements_import_ordered logements_sans_loyer=convention.lot.logements_sans_loyer_import_ordered logements_corrigee=convention.lot.logements_corrigee_import_ordered logements_corrigee_sans_loyer=convention.lot.logements_corrigee_sans_loyer_import_ordered checked=False %}
                     {% endif %}
+
                     {% if 'duree' in avenant_list %}
-                        {% include "conventions/recapitulatif/financement.html" with target_url='conventions:avenant_financement' %}
+                        {% include "conventions/recapitulatif/financement.html" with target_url='conventions:avenant_financement' checked=True %}
+                    {% else %}
+                        {% include "conventions/recapitulatif/financement.html" with target_url='conventions:avenant_financement' checked=False %}
```

Comment:
J'ai pas trouvé comment écrire checked={{ 'duree' in avenant_list }} ou quelque chose comme ça..
Je peux ajouter des nouvelles variables au contexte pour éviter de faire ce if / else, je ne suis pas sûr de ce qui est plus clair

Reply:
il y a surment moyen de faire plus propre avec un ou des blocks `with`